suite-native/module-accounts-import/src/components/DevXpub.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Box, Button } from '@suite-native/atoms';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+type DevXpubProps = {
+    symbol: NetworkSymbol;
+    onSelect: ({ xpubAddress }: { xpubAddress: string }) => void;
+};
+
+// Note: Btc and testnet are required right now. Everything else is always optional and can be undefined.
+const devXpubs: Partial<Record<NetworkSymbol, string>> & Record<'btc', string> = {
```

Comment:
Do not complicate it, it's not public API or even "production" component. Go with simple `Record<NetworkSymbol, string>`. If anyone will want to remove BTC he will probably have reason why is doing that

Reply:
https://github.com/trezor/trezor-suite/pull/6991/commits/ff586ec5e32f4d479607d13122fa0c03ab633a55